.github/workflows/ci.yaml

```diff
@@ -30,15 +30,15 @@ jobs:
             libvpx-dev \
             libx264-dev
       - name: go vet
-        run: go vet ./...
+        run: go vet `go list ./... | grep -v mmal`
```

Comment:
`$()` would be better than backquotes for safer nesting.

Reply:
Updated.